netbox/dcim/views.py

```diff
@@ -2005,11 +2019,18 @@ class DeviceDeviceBaysView(DeviceComponentsView):
 
 @register_model_view(Device, 'inventory')
 class DeviceInventoryView(DeviceComponentsView):
-    actions = ('add', 'import', 'export', 'bulk_edit', 'bulk_delete', 'bulk_rename')
     child_model = InventoryItem
     table = tables.DeviceInventoryItemTable
     filterset = filtersets.InventoryItemFilterSet
     template_name = 'dcim/device/inventory.html'
+    actions = {
+        'add': {'add'},
+        'import': {'add'},
+        'export': {'view'},
+        'bulk_edit': {'change'},
+        'bulk_delete': {'delete'},
+        'bulk_rename': {'change'},
+    }
```

Comment:
These are all duplicated (lines 1980, 2003, 2026, 2211, 2283, ...), can we make a STD_ACTIONS defined and then just do actions = STD_ACTIONS to be more DRY?